### ISSUE TITLE
Manual removal of `position: sticky` and `top: 0px` added by constructor

### ISSUE THREAD

**Issue by manuhornung** [dollarshaveclub/stickybits#446]
## Requested Update
In the case where stickybits falls back to the native `position:sticky` style, there should be a way to also have stickybits remove the inline styles again that it added. Right now, it doesn't keep track of the instances it added the inline styles to, which is why we can't call `removeInstance` on them to remove the stickiness again.
One has to manually call
```
element.style.removeProperty('top');
element.style.removeProperty('position');
```
which is an abstraction leak since now we have to know about the exact styles added by the constructor.

## Why Is This Update Needed?
In our use case, we only want the element to be sticky for a certain window size. We added a media query listener which should invoke and cleanup stickybits based on the matching of the query.

## Are There Examples Of This Requested Update Elsewhere?
I don't know

I think this issue has already been mentioned as a comment in https://github.com/dollarshaveclub/stickybits/issues/188#issuecomment-395053855

A workaround has been suggested there as well by setting `useStickyClasses` but that has other unnecessary side-effects and also feels like a hack.

I would love to clean up the abstraction here and have a way to make stickybits clean up after itself also in the case of modern browsers!

**Comment by yowainwright** [comment]
@manuhornung would you like to work on this fix/update? ~Thanks

**Comment by manuhornung** [comment]
Unfortunately I have other issues on my list with a higher priority and we went for a workaround on our userland side. I will come back to this once I find the time but please feel free to go ahead without me.

Thanks for your quick reply in any case, I'm happy that you also see this as a valuable change.

**Comment by yowainwright** [comment]
Hey @manuhornung I apologize that it took me so long to get back to you. 

Could you show the problem with this [Codepen](https://codepen.io/yowainwright/pen/gRgdep)? 

The Codepen I placed above cleans up (using `.cleanup()`) each instance and it appears to work as intended—removing inline styles. 

I am not saying that there is not an issue. I am saying I'm not quite clear on the issue now that I'm finally trying to fix it. 😃

**Comment by manuhornung** [comment]
Thanks for trying to investigate this!
First off, there seems to have been a bug in the codepen you prepared. I'm just saying this in case you're also using this in other contexts: The JS tried to apply the stickiness to `.child-1` when in fact there are only thee `.child` elements and no `.child-1`.

BTT though: I adapted the Codepen to reproduce the issue here: https://codepen.io/anon/pen/moPNeo

Our use case is that we do _not_ want to have the sticky classes on the element. That's why I removed `{ useStickyClasses: true }` from the setup. This makes stickybits fall back to a code path where it doesn't memorize the elements it modified though, which is why `.cleanup()` becomes a no-op.

I know that adding `{ useStickyClasses: true }` is a workaround for this. I think it's worth fixing this at the root though because right now, it's not possible to rely on the native fallback without applying extra classes to the element.
